crates/starknet-server/src/cli.rs

```diff
@@ -93,6 +99,11 @@ impl Args {
                 None => random_number_generator::generate_u32_random_number(),
             },
             total_accounts: self.accounts_count,
+            account_contract_class: self.account_class.get_class().expect("Invalid account class"),
```

Comment:
I know that those expects are not goint to fail, but to_starknet_config method is used only inside main function which returns Result. So error propagation should not be a problem when calling it. Thats why the return type of to_starknet_config could be changed to Result<StarknetConfig, anyhow::Error>

Reply:
Looks much better now: https://github.com/0xSpaceShard/starknet-devnet-rs/pull/196/commits/97bf1f0e5da85478dc506632dd2e703cf09dac20